Dockerfile

```diff
@@ -1,29 +1,42 @@
 ﻿# syntax=docker/dockerfile:1
+ARG DOTNET=7.0
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0 as build-env
+FROM mcr.microsoft.com/dotnet/runtime:$DOTNET AS base
+WORKDIR /publish
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
+ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
+RUN apt-get update \
+  && apt-get install -y libx11-6 libx11-xcb1 libatk1.0-0 libgtk-3-0 libcups2 libdrm2 libxkbcommon0 libxcomposite1 libxdamage1 libxrandr2 libgbm1 libpango-1.0-0 libcairo2 libasound2 libxshmfence1 libnss3 chromium \
+  && apt-get autopurge -y \
+  && apt-get autoclean -y
+
+FROM mcr.microsoft.com/dotnet/sdk:$DOTNET AS build-env
 WORKDIR /src
-COPY CLI /src/CLI
-COPY Tranga /src/Tranga
-COPY Logging /src/Logging
+
 COPY Tranga.sln /src
-RUN dotnet restore /src/Tranga/Tranga.csproj
-RUN dotnet publish -c Release -o /publish
+COPY CLI/CLI.csproj /src/CLI/CLI.csproj
+COPY Logging/Logging.csproj /src/Logging/Logging.csproj 
+COPY Tranga/Tranga.csproj /src/Tranga/Tranga.csproj
+RUN dotnet restore /src/Tranga.sln
+
+COPY . /src/
+RUN dotnet publish -c Release -o /publish -maxcpucount:1 
```

Comment:
`-maxcpucount:1` GitHub Action while building arm imagae has problems without it